src/actions/actionIndex.js

```diff
@@ -0,0 +1,105 @@
+import axios from 'axios'
+
+const APIkey = 'c905350f371fe191'
+
+export const GET_TODAYS_SUMMARY = 'GET_TODAYS_SUMMARY'
+export const getTodaysSummary = () => {
+    return function(dispatch) {
+        axios.get(`http://api.wunderground.com/api/${APIkey}/forecast/q/CA/San_Francisco.json`)
+        .then(function(res) {
+            dispatch({
+                type: GET_TODAYS_SUMMARY,
+                payload: res.data
+            })
+        })
+        .catch((e) => {console.error(413, 'Error: ', e)})
+    }
+}
+
+
+export const GET_CURRENT_WEATHER = 'GET_CURRENT_WEATHER'
+export const getCurrentWeather = () => {
+	return function(dispatch) {
+        axios.get(`http://api.wunderground.com/api/${APIkey}/hourly/q/CA/San_Francisco.json`)
+      	.then(function(res) {
+            dispatch({
+        		type: GET_CURRENT_WEATHER,
+        		payload: res.data
+        	})
+    	})
+    	.catch((e) => {console.error(412, 'Error: ', e)})
+	}
+}
+
+export const GET_HISTORY_WEATHER = 'GET_HISTORY_WEATHER'
+export const getHistoryWeather = (year) => {
+    return function(dispatch) {
+        const todaysMonth = (new Date()).getMonth() + 1
+        const todaysDate = (new Date()).getDate()
+        const tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000)
+        const tomorrowsMonth = (new Date(tomorrow)).getMonth() + 1
+        const tomorrowsDate = (new Date(tomorrow)).getDate()
+        const getMonthString = (month) => {
+            let formatedMonth = ''
+            if (month < 10) {
+                formatedMonth = '0' + month
+            } else {
+                formatedMonth = month.toString()
+            }
+            return formatedMonth
+        }
+        const getDateString = (date) => {
```

Comment:
This is the same exact code as `getMonthString`. The two could be refactored into a single generic method.